test/snapp_base.js

```diff
@@ -998,12 +996,17 @@ contract("SnappBase", async (accounts) => {
 
   describe("Larger Test Cases", () => {
     it("Fill deposit batch", async () => {
-      const instance = await MultiFlux.new()
+      const multiCaller = await MultiCaller.new()
+      const instance = await SnappBase.new()
       const core = await SnappBaseCore.new()
-      await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2)
+      // Note that only the token at index 1 is important here.
+      const token = (await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2))[1]
```

Comment:
nit: I'd just take all tokens here and register them for convenience (instead of just taking the 2nd token).

Reply:
done!